client/ayon_core/modules/deadline/abstract_submit_deadline.py

```diff
@@ -594,7 +601,11 @@ def submit(self, payload):
 
         """
         url = "{}/api/jobs".format(self._deadline_url)
-        response = requests_post(url, json=payload)
+        kwargs = {}
+        if auth:
+            kwargs["auth"] = auth
```

Comment:
I feel like the if statement isn't needed here since `requests_post` function itself does the same check? As such, we don't need the `kwargs` logic here and can just pass `auth=auth`?